### PR TITLE
GTM and GA4 implementation

### PR DESCRIPTION
**Pull request by tunetheweb**
Changes proposed in this pull request:

- Implements GTM and GA4 similar to web.dev
- Changes cookie banner implementation more inline with web.dev
- Allows storage values as well as href for announcement banners

### CODE REVIEW

File: site/_includes/layouts/base.njk [hunk 1]
```diff
@@ -17,12 +17,12 @@
...
     <link rel="preload" as="font" crossorigin href="/fonts/google-sans-v2003/medium/latin.woff2">
 
-    
+
     {% block css %}{% endblock %}
 
     <script type="module" src="{{ helpers.hashForProd('/js/main.js') }}"></script>
-    {% if process.env.NODE_ENV === 'production' %}
-      <script async src="https://www.google-analytics.com/analytics.js"></script>
+    {% if process.env.NODE_ENV !== 'production' %}
```
Comment (tunetheweb) [inline site/_includes/layouts/base.njk:24]: Reminder to myself - revert this change (which is only there for testing) before merging

Comment (tunetheweb) [reply site/_includes/layouts/base.njk:24]:
```suggestion
    {% if process.env.NODE_ENV === 'production' %}
```

**Comment by tunetheweb** [comment]
OK @matthiasrohmer  and @devnook I think this is ready for review.

Basically copies the implementation from web.dev.

**Review by devnook — APPROVED** [review]
Thanks @tunetheweb !
